### DISCUSSION TITLE
How can I downgrade the device firmware via OTA

### DISCUSSION THREAD

**Discussion by electronicm** [Koenkk/zigbee2mqtt#21632] [General]
Since the firmware upgrade 1.1.4 for the Sonoff TRVZB has been reverted in #[d05d2a0](https://github.com/Koenkk/zigbee-OTA/commit/d05d2a0b42afdf522cb90f6b1ef4ee33d05433ab), I now have to downgrade some devices back to 1.1.1.

What is the best way to do this?

I am running z2m (alpine) in lxc on a pve.

**Comment by einfachuli** [comment]
Same Problem here. As I try to downgrade with local index and url firmware or local firmware file and „force“ true parameter set, the ota process fails with invalid firmware file. So I have no solution.

**Reply by frops83** [reply]
Same Problem Here, Hope it gets solved soon

**Reply by sgaert** [reply]
here the same Problem.

**Comment by einfachuli** [comment]
It is important to make a factory reset to make normal operation possible again. Insert the batteries while the button is pressed. (Display shows FR) In this way it is working normal and  I can use it again. Firmware downgrade is still not working.

**Reply by electronicm** [reply]
Only as a source reference: 

[https://forum.ewelink.cc/t/trvzb-firmware-update-1-1-4-big-problem/29029/15](https://forum.ewelink.cc/t/trvzb-firmware-update-1-1-4-big-problem/29029/15)

https://europe1.discourse-cdn.com/standard21/uploads/ewelinkforum/original/2X/f/fd095c6fe1ffe466c93aa905af09e654592960b2.jpeg

**Comment by electronicm** [comment]
With firmware 1.1.4 and after factory reset, the thermostats no longer heat above 21°C. The set target temperature does not matter because the thermostat locks off beforehand.

![IMG_20240305_164941.jpg](https://github.com/Koenkk/zigbee2mqtt/assets/13643227/060d8e81-1598-4fe8-bfee-dd77f590db94)
I have the same behavior with all thermostats with this firmware.

**Comment by einfachuli** [comment]
Yes „normal“ operation is not as normal as as the 1.1.1 release.  I had to adjust the local temperature calibration to rise the temperature in the room to the desired level. I hope there will be a correction update. New release to update with old Firmware or new update with new Funktion. This way it goes by now (what is sonoff doing here?) I am glad I only updateted one device.
